Manifest.cs

```diff
@@ -11,6 +11,6 @@
     Id = "Etch.OrchardCore.Widgets",
     Name = "Common Widgets",
     Description = "Provides widgets commonly found on web pages.",
-    Dependencies = new[] { "OrchardCore.Widgets", "OrchardCore.Flows", "OrchardCore.Html", "Etch.OrchardCore.Blocks", "Etch.OrchardCore.Fields.CodeField", "Etch.OrchardCore.Fields.ColourField", "Etch.OrchardCore.Fields.ResponsiveMedia", "OrchardCore.Layers" },
+    Dependencies = new[] { "OrchardCore.Widgets", "OrchardCore.Flows", "OrchardCore.Html", "Etch.OrchardCore.Blocks", "Etch.OrchardCore.Fields.CodeField", "Etch.OrchardCore.Fields.ColourField", "Etch.OrchardCore.Fields.ResponsiveMedia", "OrchardCore.Layers", "OrchardCore.Title.Models" },

```

Comment:
You don't include the "Models" namespace in the dependency, only the project name (or feature name if the module provides multiple features).